src/index.js

```diff
@@ -12,18 +12,20 @@ const ToLocalStorage = (tasks) => {
 let list;
 let x = 0;
 let y = 0;
+let z = 0;
 function renderTasks() {
   list = JSON.parse(localStorage.getItem('to-do')) || [];
   document.getElementById('table').innerHTML += '';
-  list.forEach((items) => {
+  list.forEach((items, place) => {
+    items.index = place + 1;
     if (items.completed) {
-      document.getElementById('table').innerHTML += `<tr class="tr check"id=${items.index} >
-          <td><input type="checkbox" class='todo'></td><td class="desc" contenteditable="true" data-index = "${y++}" >${items.Description}
+      document.getElementById('table').innerHTML += `<tr class="tr "id=${items.index} >
+          <td><input type="checkbox" class='todo' data-index = "${z++}" checked></td><td class="desc check" contenteditable="true" data-index = "${y++}" >${items.Description}
           </td> <td><i class="fa-solid fa-ellipsis-vertical" id="${x++}"></i></td></tr>
           <tr  class="tr">`;
     } else {
       document.getElementById('table').innerHTML += `<tr class="tr"id=${items.index} >
-      <td><input type="checkbox" class='todo'></td><td class="desc" contenteditable="true" data-index = "${y++}" >${items.Description}
+      <td><input type="checkbox" class='todo' data-index = "${z++}" ></td><td class="desc" contenteditable="true" data-index = "${y++}" >${items.Description}
```

Comment:
- [ ] I will advise you to create a separate function for the index so that it can always be reset either upon adding or upon deleting.
At the moment, the index is still yet to be reset.
![image](https://user-images.githubusercontent.com/86200356/215488325-c24e8d89-2c5d-4300-9d71-ba85a71ba36d.png)